src/cli/build/build.helpers.ts

```diff
@@ -20,6 +20,24 @@ import {
 } from "../../lib/assets/k8sObjects";
 import { Reloader } from "../types";
 
+/**
+ * Check if a file exists at a given path.
+ *
+ * @param filePath - Path to the file (relative or absolute).
+ * @returns true if the file exists, false otherwise.
+ */
+export function fileExists(entryPoint: string = "pepr.ts"): string {
+  try {
+    accessSync(resolve(entryPoint), constants.F_OK);
+  } catch {
+    console.error(
```

Comment:
```suggestion
    Log.error(
```

This should probably use pino since that's our preferred logging mechanism

Reply:
ah yes good catch